### ISSUE TITLE
Add docs for interfacing with the HTTP server

### ISSUE THREAD

**Issue by Makefile-dot-in** [AndrewVos/vbar#9]
So, because vbar now uses HTTP instead of D-Bus, issue #6 was closed. However, we could still document the HTTP API.

**Comment by AndrewVos** [comment]
Absolutely. I'll wait until I'm 100% happy with the routes I'm going to use.

I might even switch away from using HTTP. Not sure. What's your feeling towards it? It seems a bit slower than using DBus in Vala, but I might be wrong.

**Comment by Makefile-dot-in** [comment]
I like HTTP so I'm more familiar with it myself, however [RPC](https://golang.org/pkg/net/rpc/) might be something to look into.
